### ISSUE TITLE
Dualshock4 Controller not recognized

### ISSUE THREAD

**Issue by ManuelGit-Hub** [The-Demp/DualSenseAPI#2]
The library makes a good impression for me, however my controller is not recognized, although my Pc recognizes it correctly. Does anyone have the same problem or know how to solve it?

**Comment by BadMagic100** [comment]
Are you connected over Bluetooth? I had some issues where the data packets being sent over Bluetooth were not always consistent. I was never able to nail down the root cause unfortunately

More info here https://github.com/BadMagic100/DualSenseAPI/blob/master/DualSenseAPI/State/DualSenseInputState.cs#L105

**Comment by ManuelGit-Hub** [comment]
Thanks for your fast reply. I am connected via the cable.

I have to admit I made a mistake. I read the ps4 controller is named Dual**Shock** and not Dual**Sense**. If I am correct the library is not intended to work with the ps4 Dual**Shock** controller. Is this right?

**Comment by BadMagic100** [comment]
That is correct, dualsense is the ps5 controller and this library exists mainly to interact with the advanced haptic features which are not made readily available by device drivers as with previous iterations of the playstation controller. No ps4 support is intended
